sbi/inference/base.py

```diff
@@ -590,7 +603,7 @@ def simulate_for_sbi(
             from.
         num_simulations: Number of simulations that are run.
         num_workers: Number of parallel workers to use for simulations.
-        simulation_batch_size: Number of parameter sets that the simulator
+        sim_batch_size: Number of parameter sets that the simulator
             maps to data x at once. If None, we simulate all parameter sets at the
             same time. If >= 1, the simulator has to process data of shape
             (simulation_batch_size, parameter_dimension).
```

Comment:
I find this description a bit confusing. maybe better:
```suggestion
        sim_batch_size: Number of parameter sets of shape
            (simulation_batch_size, parameter_dimension) that the simulator receives
            per call. If None, we set simulation_batch_size=num_simulations and 
            simulate all parameter sets with one call. Otherwise, we construct
            batches of parameter sets and distribute them among num_workers.
```

Reply:
Yes it was not completely accurate (did not consider the case where `sim_batch_size > num_simulations`). I edited the comment including your suggestion.